src/scripts/dungeons/Dungeon.ts

```diff
@@ -363,6 +363,17 @@ dungeonList['Sky Pillar'] = new Dungeon('Sky Pillar',
     ],
     40000, 101, 5);
 
+dungeonList['Sealed Cave'] = new Dungeon('Sealed Cave',
+    ['Zubat', 'Poochyena', 'Nosepass'],
+    [GameConstants.BattleItemType.xAttack, GameConstants.BattleItemType.Lucky_egg],
+    400000,
+    [
+        new DungeonBossPokemon('Regirock', 1200000, 20),
```

Comment:
Is Regirock supposed to be weaker than the other 2?

Reply:
Fixed